app/assets/javascripts/questions.js

```diff
@@ -26,7 +26,21 @@ function initEditQuestion() {
   $('#question_text, #question_answers_attributes_0_text').keyup(validateEditQuestionInput);
 
   // track qset user created a question in
-  $('#question_qset_id').change(function() {
+  $('#qset_id_input').change(function() {
     Cookies.set('new_question_qset_id', this.value, { expires: 1000 });
   });
+
+  hasQuestionsIsSelectable();
+
+}
+
+// Closes modal, displays qset name next to Choose Qset button, and populates hidden qset-id input
+function hasQuestionsIsSelectable() {
+  $('.qset-selectable').click(function() {
+    $('.qset-display').text($(this).find('.qset-name-modal').text());
+    $('#qset_id_input').val($(this).find('.qset-id-modal').text());
+    $('#myModal').modal('hide');
+
+    validateEditQuestionInput();
+  });
 }
```

Comment:
I think by our style guidelines we always leave a line at the bottom of each page